includes/Core/Authentication/Authentication.php

```diff
@@ -547,52 +603,65 @@ private function inline_js_admin_data( $data ) {
 			$data['userData']['picture'] = $profile_data['photo'];
 		}
 
-		$client_data = $this->credentials->get();
-		$apikey      = $this->get_api_key_client()->get_api_key();
-		$gcp_project = $this->gcp_project->get();
+		$auth_client = $this->get_oauth_client();
+		if ( $auth_client->using_proxy() ) {
+			$access_code                 = (string) $this->user_options->get( Clients\OAuth_Client::OPTION_PROXY_ACCESS_CODE );
+			$data['proxySetupURL']       = esc_url_raw( $auth_client->get_proxy_setup_url( $access_code ) );
```

Comment:
```suggestion
			$data['proxySetupUrl']       = esc_url_raw( $auth_client->get_proxy_setup_url( $access_code ) );
```

Reply:
Why the case change? There are varying opinions on what is the better way to write acronyms, but we've been using mostly `URL` throughout, so I'd prefer to do so.

Reply:
I was of the understanding that we were going to standardize on the `*Url` variant (See the IB for #449) - there are several instances of it even in this PR (`connectUrl`, `disconnectUrl`, etc).

It really doesn't matter too much either way I think but we should standardize on one and stick with it which was my intention here. We have a lot of usage in both forms.

I think Gutenberg uses the all-caps version, so that might be the way to go if we want to be more consistent with core WP.

Reply:
Ah, that's fair. I'd say let's keep this as is for now and decide on one approach early next week so we can make the necessary replacements before launch.